### PR TITLE
Do not use colon in archived log files

### PR DESCRIPTION
**Pull request by mhajas**
Any idea for a nicer fix? I think it is important to have the date and time there.

### CODE REVIEW

File: provision/aws/rosa_create_cluster.sh [hunk 1]
```diff
@@ -56,14 +56,18 @@ fi
 
 mkdir -p "logs/${CLUSTER_NAME}"
 
+function custom_date() {
+    echo "$(date -uIs | sed -e "s/:/-/g")"
```
Comment (ahus1) [inline provision/aws/rosa_create_cluster.sh:60]: You might be looking for something like this:

```suggestion
    echo "$(date '+%Y%m%d-%H%M%S')"
```

Same below for the other date.

**Review by ahus1 — CHANGES_REQUESTED** [review]
See comment below for a formatting of a date which is a bit different.

File: provision/aws/rosa_create_cluster.sh (hunk 1 above)
Comment (mhajas) [reply provision/aws/rosa_create_cluster.sh]: Thanks, done

**Review by ahus1 — APPROVED** [review]
LGTM!

**Pull request merged**